apps/crn-frontend/src/network/teams/TeamOutput.tsx

```diff
@@ -63,16 +63,16 @@ const TeamOutput: React.FC<TeamOutputProps> = ({ teamId }) => {
   );
   const team = useTeamById(teamId);
   const [errors, setErrors] = useState<ValidationErrorResponse['data']>([]);
-  const { isEnabled } = useFlags();
+
+  const { canCreate } = useContext(ResearchOutputPermissionsContext);
```

Comment:
Why do we need to wrap the components with a Context? I can understand it if it's deeply nested but in this case we can access state directly.

Reply:
That was @kmaid suggestion :)